backend/schemas/source/excel/libs/InstructionsCorrectiveActionPlan.libsonnet

```diff
@@ -0,0 +1,17 @@
+{
+  reference_number: {
+    instructions: [
```

Comment:
Workbook 2 is labelled Notes to SEFA 
![image](https://github.com/GSA-TTS/FAC/assets/26260777/c77b85f2-0f59-4a6f-bf4c-67509f1ea1eb)